tests/backup/backup_helper.go

```diff
@@ -1337,3 +1393,87 @@ func CreateCustomRestoreWithPVCs(restoreName string, backupName string, namespac
 	}
 	return deployment.Name, nil
 }
+
+// GetOrdinalScheduleBackupName returns the name of the schedule backup at the specified ordinal position for the given schedule
+func GetOrdinalScheduleBackupName(ctx context.Context, scheduleName string, ordinal int, orgID string) (string, error) {
+	allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+	if err != nil {
+		return "", err
+	}
+	if len(allScheduleBackupNames) == 0 {
+		return "", fmt.Errorf("no backups were found for the schedule [%s]", scheduleName)
+	}
+	if ordinal < 1 {
```

Comment:
Can we make this as the first check as if the ordinal value sent is less that 1 , it does not make sense to continue any further.
You have also added "if ordinal > len(allScheduleBackupNames)" so if ordinal value provided is beyond  len(allScheduleBackupNames), this case is also taken care.

Reply:
If the ordinal value is less than 1, I am using the length of allScheduleBackupNames to indicate the valid range of values in the log.

Reply:
Made sure to check ordinal value before enumerating backups.